src/plugins/LADTable/pluginSpec.js

```diff
@@ -28,29 +28,23 @@ import {
     getLatestTelemetry
 } from 'testUtils';
 
+const TABLE_BODY_ROWS = '.js-lad-table__body__row';
```

Comment:
I'm expecting to see these classes being used in the markup, but I don't see them. What am I missing here?

Reply:
Should be good now!